app/controllers/users_controller.rb

```diff
@@ -0,0 +1,48 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to @user
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+    @mode = 'edit'
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to @user
+    else
```

Comment:
- [ ] Kindly if the user gets saved, it should redirect to the `edit_user_path(<USER_ID>)`. This is a hard requirement for this project

Reply:
Add changes like requested however update was not a part of the assignment ...